packages/runner-ct/src/app/ReporterHeader.module.scss

```diff
@@ -0,0 +1,5 @@
+@use '../variables' as *;
+
+.ctReporterHeader { 
+  min-height: $runner-ct-header-height;
```

Comment:
@lmiller1990 it looks like this line of code can conflict with your PR

Reply:
Probably, I'll deal with it. Thanks for the notice.